# nav_msgs/Path trajectory
string tree
---
